src/sage/rings/integer.pyx

```diff
@@ -7163,6 +7163,45 @@ cdef class Integer(sage.structure.element.EuclideanDomainElement):
         else:
             raise ValueError("algorithm must be one of: 'pari' or 'gmp' (alias: 'mpir')")
 
+    def to_bytes(self, length=1, byteorder="big", is_signed=False):
+        """
+        Return an array of bytes representing an integer.
+
+        Internally relies on the python ``int.to_bytes()`` method.
+
+        INPUT:
+
+        - ``length`` -- positive integer (default: ``1``); integer is represented in
+          ``length`` bytes. ``OverflowError`` is raised if the integer is not representable
+          with the given length.
```

Comment:
```suggestion
        - ``length`` -- positive integer (default: ``1``); integer is represented in
          ``length`` bytes
```
The `OverflowError` portion is not part of the input. I don't think its needed, but if you want to add it, you can add it before the `INPUT:` block.